src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -189,6 +188,46 @@ private ProcessResult tryCreateNewCase(
                 log.error(msg);
                 throw new MultipleCasesFoundException(msg);
             }
+
+            if (result.caseId == null) {
+                return new ProcessResult(result.warnings, result.errors);
+            } else {
+                return tryPublishPaymentMessageAndFinalise(
+                    configItem.getService(),
+                    exceptionRecordData,
+                    result.caseId
+                );
+            }
+        }
+    }
+
+    private ProcessResult tryPublishPaymentMessageAndFinalise(
+        String serviceName,
+        CaseDetails exceptionRecordData,
+        long caseId
+    ) {
+        try {
+            paymentsProcessor.updatePayments(exceptionRecordData, caseId);
+
+            return new ProcessResult(
+                exceptionRecordFinalizer.finalizeExceptionRecord(
+                    exceptionRecordData.getData(),
+                    caseId,
+                    CcdCallbackType.CASE_CREATION
+                )
+            );
+        } catch (PaymentsPublishingException exception) {
+            log.error(
+                "Failed to send update to payment processor for {} exception record {}",
+                serviceName,
+                exceptionRecordData.getId(),
+                exception
+            );
+
+            return new ProcessResult(
+                emptyList(),
+                singletonList("Payment references cannot be processed. Please try again later")
```

Comment:
worth confirming this message with Ade as this will be displayed on UI.

Reply:
it was confirmed long since. same message is used for attaching docs callback